src/App.tsx

```diff
@@ -15,49 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Goods = string;
+type Reverse = boolean;
+
+enum Sortfield {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
```

Comment:
enum's items shoud be in CamelCase 
```suggestion
enum Sortfield {
  Alphabet = 'alphabet',
  Length = 'length',
  Default = '',
}
```